spec/std/isa/csr/Smaia/mtopei.yaml

```
@@ -0,0 +1,45 @@
+# Copyright (c) 2025 Kallal Mukherjee
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: mtopei
+long_name: Machine Top External Interrupt
+address: 0x35C
+priv_mode: M
+length: MXLEN
+writable: true
+description: |
+  Provides information about the highest-priority pending-and-enabled external interrupt
+  for machine level, with claim-and-clear semantics on write.
+definedBy: Smaia
+fields:
+  IID:
+    location: 26-16
+    long_name: Interrupt Identity
+    description: |
+      Identity of the highest-priority pending-and-enabled external interrupt.
+
+      When written with a non-zero value, clears the corresponding IMSIC interrupt.
+    type: RW-R
+    reset_value: UNDEFINED_LEGAL
+    definedBy: Smaia
+    sw_write(csr_value): |
+      if (csr_value.IID != 0) {
+        if (implemented?(ExtensionName::Smaia)) {
+          clear_imsic_interrupt(PrivilegeMode::M, csr_value.IID);
+        }
+      }
```

Let's simplify here:
```suggestion
      if (csr_value.IID != 0 && implemented?(ExtensionName::Smaia)) {
        clear_imsic_interrupt(PrivilegeMode::M, csr_value.IID);
      }
```